hadolint.cabal

```diff
@@ -3,8 +3,6 @@ cabal-version: 2.0
 -- This file has been generated from package.yaml by hpack version 0.34.2.
 --
 -- see: https://github.com/sol/hpack
---
--- hash: 37f589477154f969b62b2232bb0d1b63a8261cdc7c3b6099f0dcaad7240bc2e7
```

Comment:
This line should not be deleted. Let stack generate this file for you automatically

Reply:
Hey @lorenzo  - I did not delete the hash manually. This seems to be the default behavior for hpack 0.34.2 
https://hackage.haskell.org/package/hpack-0.34.0/changelog

For now, I will downgrade my stack version to use the 0.33 hpack so that the hash is still generated. But its bound to come our way.

Reply:
I see, did not know a new version was out. What version where you using?

Reply:
I dont remember the exact stack version. But the hpack version was 0.34.2 which has the default of no hash strategy.